src/api.ts

```diff
@@ -380,7 +380,7 @@ export interface ToolsRequest {
 }
 
 export type TokensRequest = {
-  chains?: ChainId[]
+  chains?: number[] | string[]
```

Comment:
Good point, but two comments:

1. Why not `ChainId` instead of `number` and `ChainKey` instead of string? We can't deal with chains that we don't know about anyway.
2. It should rather be `(ChainId | ChainKey)[]` I think, this way the type would also allow `[1, OPT]` and I see no reason not to allow that

Reply:
Applied